build.cake

```diff
@@ -60,7 +60,8 @@ Task("VerifyCodeGen")
 	.Does(() => CodeGen(verify: true));
 
 Task("Test")
-	.IsDependentOn("VerifyCodeGen")
+	.IsDependentOn("Build")
+//	.IsDependentOn("VerifyCodeGen")
```

Comment:
Fair, as long as it gets run during automated builds. Add to `SourceIndex`?

Reply:
Accidental commit.